backend/server.go

```diff
@@ -5,6 +5,7 @@ import (
 	"api/config"
 	"api/db"
 	"api/helpers"
+	authMiddleware "api/middleware"
```

Comment:
Does this need to be aliased? Why wouldn't we call it `api/authMiddleware` if you feel the need to alias it

Reply:
It's aliased because it conflicts with `github.com/go-chi/chi/v5/middleware`.  I don't mind changing it to `api/authMiddleware`.